@attr(tags=['advanced', 'selfservice'], required_hardware='false')
def test_02_dedicate_vlan_range_invalid_account(self):
    'Dedicate a guest vlan range to invalid account\n\n        # Validate the following:\n        # 1. Create an account in root domain\n        # 2. Update physical network with new guest vlan range\n        # 3. Try to dedicate it to invalid account\n        # 4. The operation should fail\n        '
    self.account = Account.create(self.apiclient, self.testdata['account'], domainid=self.domain.id)
    self.cleanup.append(self.account)
    new_vlan = ((self.physical_network.vlan + ',') + self.free_vlan['partial_range'][0])
    self.physical_network.update(self.apiclient, id=self.physical_network.id, vlan=new_vlan)
    with self.assertRaises(Exception):
        PhysicalNetwork.dedicate(self.apiclient, self.free_vlan['partial_range'][0], physicalnetworkid=self.physical_network.id, account=(self.account.name + random_gen()), domainid=self.account.domainid)
    return
